components/AnimatedRegion.js

```diff
@@ -1,45 +1,50 @@
 /* eslint-disable */
+import {Animated} from 'react-native';
 
-class AnimatedRegion extends AnimatedWithChildren {
-  // latitude: AnimatedValue;
-  // longitude: AnimatedValue;
-  // latitudeDelta: AnimatedValue;
-  // longitudeDelta: AnimatedValue;
-  // _listeners: {[key: string]: {
+const AnimatedWithChildren = Object.getPrototypeOf(Animated.ValueXY);
```

Comment:
Well, it's not really that sneaky :) There's prototype chains in Javascript, so why not use them. Ideally react-native would allows us to create our own custom composition classes, but until that is the case we have to be a bit creative. This is quite solid, but I do think we should add a check to ensure that the parent prototype of Animated.ValueXY is indeed AnimatedWithChildren. I'll try to add some code that does this, so in case it breaks we'll immediately know.